HISTORY.rst

```diff
@@ -2,6 +2,38 @@
 History
 =======
 
+3.39.0 (2021-07-08)
+-------------------
+Critical Changes:
+* A new Snowfakery task with better usability and multi-processor support. Look at the CumulusCI docs to learn the new syntax.
+* Snowfakery 2.0 with various new features, especially the ability to query into orgs.
+
+Changes:
+
+
+* Add tests to increase coverage of Snowfakery to 100%. (#2711)
+* Also remove dead code and a few other stylistic improvements:
+ * e.g. get_debug_mode() is captured just once as an instance attribute.
+ * dead code is removed in some places.
+ * a few assertion or warning branches are covered with pragmas. (#2711)
+* Improved our robot documentation so that it's possible to link to keyword documentation instead of having to download it locally (#2696)
```

Comment:
```suggestion
* We improved our Robot documentation so that it's possible to link to keyword documentation instead of having to download it locally (#2696)
```